src/main/steps/index.ts

```diff
@@ -76,7 +76,8 @@ export type StepWithContent = ({ generateContent: TranslationFn; form: FormConte
 export const stepsWithContent = ((): StepWithContent => {
   const results: StepWithContent = [];
   for (const step of sequence) {
-    const content = require(`${__dirname}${step.url}/content.ts`);
+    const stepContentFile = `${__dirname}${step.url}/content.ts`;
+    const content = fs.existsSync(stepContentFile) ? require(stepContentFile) : {};
```

Comment:
is this check for the empty application submitted page?

Reply:
Yes, this is for steps that don't have content and handle logic only like the `/payment_callback` and the `/application-submitted` endpoints.